### PR TITLE
Updated API

### PR DESCRIPTION
**Pull request by beliarh**
Deleted old files that was causing `"WARNING: document isn't included in any toctree"` (see #729).
Warnings like ` "WARNING: unknown document"` (references to deleted files) will be fixed in next patches.
Some `@param` was removed (from cphalcon docblocks), so type of this parameters is `*uknown*` for script now.
It will also be fixed in next patches.

### CODE REVIEW

File: en/api/Phalcon_Annotations_Annotation.rst [hunk 1]
```diff
@@ -60,13 +60,13 @@ Returns an argument in a specific position
...
 
-public *mixed*  **getNamedArgument** (*string* $name)
+public *mixed*  **getNamedArgument** (*unknown* $name)
 
 Returns a named argument
 
 
 
-public *mixed*  **getNamedParameter** (*string* $name)
+public *mixed*  **getNamedParameter** (*unknown* $name)
```
Comment (sergeyklay) [inline en/api/Phalcon_Annotations_Annotation.rst]: Why?

**Comment by sergeyklay** [comment]
> Some @param was removed (from cphalcon docblocks), so type of this parameters is *uknown* for script now.

It's bad

File: en/api/Phalcon_Annotations_Annotation.rst (hunk 1 above)
Comment (beliarh) [inline en/api/Phalcon_Annotations_Annotation.rst]: https://github.com/phalcon/cphalcon/blob/master/phalcon/annotations/annotation.zep#L206-L214
```zephir
/**
 * Returns a named parameter
 *
 * @return mixed
 */
public function getNamedParameter(string! name)
{
	return this->getNamedArgument(name);
}
```
Removed in phalcon/cphalcon@379e45d.
See also phalcon/cphalcon@af3f7b65 (annotations docblock cleanup).

**Comment by beliarh** [comment]
Improved gen-api script.
Fixed some `*unknown*` types:
* classes like `Phalcon\\...` (`$parameter->getClass()->name`)
* arrays (`$parameter->isArray()`)

But I dont know how to fix `*unknown*` for strings.

**Comment by SidRoberts** [comment]
Thanks. The Zephir project has a Documentation namespace which is able to get parameter/return types from doc blocks and Zephir code but I've never had the time to implement it. I think that's the most reliable way to get this information.

**Pull request merged**